charts/vault/templates/vault-rb.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.plugins.readKuberenetsSecrets.namespaces }}
+{{- $releaseNamespace := .Release.Namespace -}}
+{{- range $ns := .Values.plugins.readKuberenetsSecrets.namespaces }}
```

Comment:
Are you sure it's the right syntax for capturing the values from the list? (I'm only aware of capturing both index and value, or a bare range with values captured in `.`.  Just verifying.

Reply:
Please see the section "Iterating over a .Values List" in https://kb.novaordis.com/index.php/Helm_Template_range.

Reply:
It shows what I meant in `or a bare range with values captured in .` 
Anyway, just test that it works, in go it would capture indexes but maybe in go templates it's different.

Reply:
yes, it works I tested it before sending the pr :-)

Reply:
Awesome, thanks. Indeed https://golang.org/pkg/text/template/  says:

> Note that if there is only one variable, it is assigned the element; this is opposite to the convention in Go range clauses.